modules/aws-eks-addon/aws-ebs-csi-driver/local.tf

```diff
@@ -0,0 +1,17 @@
+
+locals {
+  default_aws_ebs_csi_driver_config = {
+    addon_name               = "aws-ebs-csi-driver"
+    addon_version            = "v1.4.0-eksbuild.preview"
+    service_account          = "ebs-csi-controller-sa"
+    resolve_conflicts        = "OVERWRITE"
+    namespace                = "kube-system"
+    additional_iam_policies  = []
+    service_account_role_arn = ""
+    tags                     = {}
+  }
+  aws_ebs_csi_driver_config = merge(
```

Comment:
rename to add_on_config?

Reply:
This can be changed to  `add_on_config` since its local to that Addon

Reply:
updated